credsweeper/rules/config.yaml

```diff
@@ -162,7 +162,7 @@
   severity: medium
   type: pattern
   values:
-  - (?P<value>eyJ[A-Za-z0-9-_=]+(\.[A-Za-z0-9-_.+\/=]+)?)
+  - (?<![0-9a-zA-Z_-])(?P<value>eyJ[A-Za-z0-9-_=]+\.eyJ[A-Za-z0-9-_=]+(\.[A-Za-z0-9-_=]+)?)
```

Comment:
JWT with a single `eyJ` entry might still be a valid credential

https://github.com/Samsung/CredSweeper/pull/104

Reply:
`?<![0-9a-zA-Z_-])` probably should not be in JWT regular expression itself
We can concatenate it to ALL pattern based expressions, so it would fix all other short expressions

Reply:
'''(?<![0-9a-zA-Z_-])''' removes case when eyJ is only a substring in long base64 case. even  ```(?<![0-9a-zA-Z])``` might reduce suspicious strings.